src/Xamarin.Android.Tools.AndroidSdk/Properties/Resources.resx

```diff
@@ -117,12 +117,23 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+
+  <data name="FailedToChangeFileSystemOwnership" xml:space="preserve">
+    <value>Failed to change ownership of filesystem object(s).</value>
```

Comment:
`object(s)` is at least sometimes translated as if it were just `objects`. For example, `item(s)` was translated as the plural `elementos` in the following example from [dotnet/msbuild](https://github.com/dotnet/msbuild/blob/ca44138662e3aa90eb9305dd31d906ef02e962cb/src/Tasks/Resources/xlf/Strings.es.xlf#L588-L589):

```xml
<source>MSB3094: "{2}" refers to {0} item(s), and "{3}" refers to {1} item(s). They must have the same number of items.</source>
<target state="translated">MSB3094: "{2}" hace referencia a {0} elementos y "{3}" hace referencia a {1} elementos. Deben tener el mismo número de elementos.</target>
```

Although for some target languages there won't be any way to improve this because the correct grammar requires multiple different wordings for different numbers of items, a small improvement for at least some target languages could be to use the phrase "one or more":

```
Failed to change the ownership of one or more filesystem objects.
```

On a different topic, this message doesn't yet suggest a resolution action, but from what I could find, `FixOwnership()` isn't called by any Xamarin.Android build tasks, so it's not currently expected that Xamarin.Android users will see this message during a build anyway. Considering that, this message could optionally be omitted from localization for now, but it's OK to include it too.

Reply:
Actually, considering that we "know" that this assembly is used by *an installer*, but we *don't* know if said installer will actually capture & redistribute the satellite assemblies containing the translated strings, it's probably better if we *don't* translate this message.